packages/clarity-js/src/data/upload.ts

```diff
@@ -124,7 +124,7 @@ function stringify(encoded: EncodedPayload): string {
     return encoded.p.length > 0 ? `{"e":${encoded.e},"a":${encoded.a},"p":${encoded.p}}` : `{"e":${encoded.e},"a":${encoded.a}}`;
 }
 
-function send(payload: string, sequence: number, last: boolean): void {
+function send(payload: string, sequence: number, beacon: boolean): void {
```

Comment:
>beacon [](start = 49, length = 6)

Suggest you do this for all 4xx rather than just 400.

Reply:
Done